internal/core/src/config/ConfigKnowhere.cpp

```diff
@@ -77,6 +77,13 @@ KnowhereSetSimdType(const char* value) {
     }
 }
 
+void
+KnowhereInitGPU(const int32_t gpu_id, const int32_t res_num) {
+#ifdef MILVUS_GPU_VERSION
```

Comment:
This looks weird to me:
1. I guess we need flag called `WITH_GPU` or `ENABLE_GPU`.
2. If `WITH_GPU`, what we should do is not making `KnowhereInitGPU` empty but never call it.

Reply:
Because in indexnode and querynode, this API will be called via cgo whatever in GPU mode or CPU mode.

Reply:
1. Can we rename it to `WITH_GPU` or `ENABLE_GPU`?
2. Do we have any way to don't call `KnowhereInitGPU` in both query/index node in go code when `WITH_GPU` disabled?